point-of-sale/src/server/core/connection.ts

```diff
@@ -1,4 +1,4 @@
 import { Connection } from '@solana/web3.js';
 import { CLUSTER_ENDPOINT } from './env';
 
-export const connection = new Connection(CLUSTER_ENDPOINT || 'https://api.devnet.solana.com', 'confirmed');
```

Comment:
👍

Reply:
@jordansexton  you can approve this merge quest 🥲